### PR TITLE
Allow replacing comment timestamp

### PR DESCRIPTION
**Pull request by alinhle**
## What does this PR do?
New slots `commentTimestamp` and `historyCommentTimestamp` to allow replacing the comment date e.g. _10 minutes ago_ with a custom component

## How do I test this PR?

- Add components to the above slots to a new or existing plugin and see it replacing the timestamp on the comment stream and comment history.

### CODE REVIEW

**Comment by kgardnr** [comment]
I'm good with this once @okbel gives it a thumbs up!

File: client/coral-embed-stream/src/components/Comment.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import TagLabel from 'talk-plugin-tag-label/TagLabel';
-import PubDate from 'talk-plugin-pubdate/PubDate';
```
Comment (okbel) [inline client/coral-embed-stream/src/components/Comment.js:5]: I'm glad we are removing this. `talk-plugin...`

File: client/coral-framework/components/CommentTimestamp.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+const CommentTimestamp = ({className, created_at}) => (
+  <div className={cn(className, styles.timestamp, 'talk-comment-timestamp')}>
+    {timeago(created_at)}
+  </div>
+);
+
+CommentTimestamp.propTypes = {
+  className: PropTypes.string,
+  created_at: PropTypes.string,
```
Comment (okbel) [inline client/coral-framework/components/CommentTimestamp.js:15]: I'm not going to bring up the casing again 😛

**Comment by okbel** [comment]
Code looks good. I'm testing it right now.

**Review by okbel — APPROVED** [review]

File: client/coral-framework/components/CommentTimestamp.js (hunk 2 above)
Comment (alinhle) [reply client/coral-framework/components/CommentTimestamp.js:15]: oh well..

Comment (alinhle) [reply client/coral-framework/components/CommentTimestamp.js:15]: This one should be camel case I guess.. >.<

**Pull request merged**